portfolio/src/main/webapp/scripts/youtube/client.js

```diff
@@ -0,0 +1,13 @@
+const baseURL = "https://www.googleapis.com/youtube/v3";
+const API_KEY = "AIzaSyBENDSXuXOOLyByHnGTzMrYRhzWHIMnIsY";
```

Comment:
Hm, possible to put this in a hidden file you don't upload to GitHub and just keep it locally for now?

Reply:
Sure! I moved this into a secret.js which I'm gitignoring.

Reply:
Awesome!